index.js

```diff
@@ -15,6 +15,62 @@
 const Bindings = require('bindings')('nodemgclient');
 const pjson = require('./package.json');
 
+// The purpose of create functions is to simplify creation of Memgraph specific
+// data types, e.g. temporal types.
+
+/**
+  * Create Memgraph compatible date object.
+  * @param {number} days - The number of days since 1970-01-01 (unix epoch).
+  */
+function createMgDate(days) {
+  return {
+    "objectType": "date",
+    "days": days,
+  }
+}
+
+/**
+  * Create Memgraph compatible local time object.
+  *
+  * @param {number} nanoseconds - The number of nanoseconds since midnight.
+  */
+function createMgLocalTime(nanoseconds) {
+  return {
+    "objectType": "local_time",
+    "nanoseconds": nanoseconds,
+  }
+}
+
+/**
+  * Create Memgraph compatible local date time object.
+  * @param {number} seconds - The number of seconds since 1970-01-01 (unix epoch).
```

Comment:
and the local_time!

Reply:
What do you exactly mean here? This: `@param {number} seconds - The number of seconds since 1970-01-01T00:00:00 (Unix epoch) and the local_time.`? If so, is that correct actually because, all time is expressed with the 0 offset?

Reply:
I mean that if you have the date `1970-01-01T00:00:01` this is `0 days` since the unix epoch + `00:00:01` (1 second since the unix epoch). Hence, you will receiver 0 (days in seconds) + 1 (second the time point) = 1 second since the unix epoch

Reply:
Aha, if I understand correctly, this is now an opposite discussion to the one about the duration :smile: Isn't this the implementation detail? A user doesn't care about how is that serialized, the user has the number of seconds, and that has to be explained?